analysis/descriptives/venn_diagram.R

```diff
@@ -223,6 +223,17 @@ venn_output <- function(cohort_name){
   
   # Save summary file ----------------------------------------------------------
   
+  # Merge any cells <= 5 to the highest cell (excluding totals) -------------
+  
+  colnamesorder <- colnames(df)
+  a <- df[-c(1,9:12)]
```

Comment:
equivalent to a<-df[,2:8] # to keep only_*, snomed_hes, snomed_death, hes_death and snomed_hes_death